server-manager.proto

```diff
@@ -237,33 +214,31 @@ message InspectStateResponse {
     optional bytes exception_data = 5; // Exception payload when finished with EXCEPTION
-    repeated Report reports = 6; // Reports produced while processing the query
+    repeated Output reports = 6; // Reports produced while processing the query
 }
 
 // Validity proof for an output
 message OutputValidityProof {
     uint64 input_index_within_epoch = 1; // Local input index within the context of the related epoch
     uint64 output_index_within_input = 2; // Output index within the context of the input that produced it
     CartesiMachine.Hash output_hashes_root_hash = 3; // Merkle root of all output hashes of the related input
-    CartesiMachine.Hash vouchers_epoch_root_hash = 4; // Merkle root of all voucher hashes of the related epoch
-    CartesiMachine.Hash notices_epoch_root_hash = 5; // Merkle root of all notice hashes of the related epoch
-    CartesiMachine.Hash machine_state_hash = 6; // Hash of the machine state claimed for the related epoch
-    repeated CartesiMachine.Hash output_hash_in_output_hashes_siblings = 7; // Proof that this output hash is in the output-hashes merkle tree. This array of siblings is bottom-up ordered (from the leaf to the root).
-    repeated CartesiMachine.Hash output_hashes_in_epoch_siblings = 8; // Proof that this output-hashes root hash is in epoch's output merkle tree. This array of siblings is bottom-up ordered (from the leaf to the root).
+    CartesiMachine.Hash outputs_epoch_root_hash = 4; // Merkle root of all outputs_epoch_root_hash for inputs in epoch
+    CartesiMachine.Hash machine_state_hash = 5; // Hash of the machine state claimed for the related epoch
+    repeated CartesiMachine.Hash output_hash_in_output_hashes_siblings = 6; // Proof that this output hash is in the output-hashes merkle tree. This array of siblings is bottom-up ordered (from the leaf to the root).
+    repeated CartesiMachine.Hash output_hashes_in_epoch_siblings = 7; // Proof that this output-hashes root hash is in epoch's output merkle tree. This array of siblings is bottom-up ordered (from the leaf to the root).
 }
 
-enum OutputEnum {
-    VOUCHER = 0;
-    NOTICE = 1;
+message InputRange {
+    uint64 first_input; // index of first input in range
+    uint64 last_input; // index of last input in range
 }
 
 // Data that can be used as proof to validate notices and execute vouchers on the base layer blockchain
 message Proof {
     uint64 input_index = 1; // Index of input starting from genesis
     uint64 output_index = 2; // Index of output (voucher or notice) in the context of the input
-    OutputEnum output_enum = 3; // Type of the output
-    OutputValidityProof validity = 4; // Validity proof for an output
-    bytes context = 5; // Data that allows the validity proof to be contextualized within submitted claims. Currently, the context is the epoch number as a ABI-encoded uint256.
+    OutputValidityProof validity = 3; // Validity proof for an output
```

Comment:
This is a proof for a specific output that has been mentioned above, so "the" output.